plugins/sampledata/testing/testing.php

```diff
@@ -999,7 +999,7 @@ public function onAjaxSampledataApplyStep4()
 			),
 			array(
 				'catid'    => $catIdsLevel5[0],
-				'tags'     => array($tagIds[0], $tagIds[1], $tagIds[2]),
+				'tags'     => array_map('strval', array($tagIds[0], $tagIds[1], $tagIds[2])),
```

Comment:
```suggestion
				'tags'     => array_map('strval', array_slice($tagIds, 0, 3)),
```
The array theoretically could be empty because empty array used as default when getting from user state.